src/srtparser/parse.js

```diff
@@ -3,7 +3,7 @@ import ParseError from './parseerror';
 import { toMS } from './date';
 
 const EOL = /\r?\n/;
-const TIME_STAMP_REGEX = /(\d+):(\d{1,2}):(\d{1,2})[,\.](\d{1,3})/;
+const TIME_STAMP_REGEX = /^\s*(\d{2}):(\d{2}):(\d{2}),(\d{3})\s*$/;
```

Comment:
What was the reasoning behind allowing whitespace before and after the timestamp? I'm not sure I agree with this change - it makes this valid:

`                    00:00:00,000->00:00:00,001           
Hello!`


Did you intend to just match the newline character, instead of all whitespace?

Reply:
I don't need to match the whitespace. I think I had it in there when I was still writing it out by hand and testing.

`00:00:00,000->00:00:00,001 Hello!` wouldn't match though because of the non-whitespace character before the EOL at `$`.

![image](https://user-images.githubusercontent.com/6240821/52589463-de906100-2e0c-11e9-8d9a-2c3f3450efec.png)

Either way, the whitespace is definitely not necessary. I'm going to get rid of it.